src/scores.ts

```diff
@@ -43,13 +43,14 @@ async function getVotes(proposalId: string): Promise<any[] | undefined> {
  * This should be implemented in snapshot.js, leading to either a breaking change or a new
  * function, e.g. named getFullScores while getScores still returns just the scores prop.
  */
+const scoreApiEndpoint = process.env.SCORE_API_URL || 'https://score.snapshot.org';
```

Comment:
Shouldn't this be named `scoreApiURL`, like the rest ?

Reply:
fixed